docs/deploy.md

```diff
@@ -199,6 +199,10 @@ spark.sql.adaptive.skewJoin.enabled true
 spark.shuffle.sort.io.plugin.class org.apache.spark.shuffle.celeborn.CelebornShuffleDataIO
 # Required Spark version >= 3.4.0, highly recommended to disable
 spark.dynamicAllocation.shuffleTracking.enabled false
+
+# Support ShuffleManager defined in user-defined jar
+# Required Spark version < 4.0.0, highly recommended to false for ShuffleManager in user-defined jar specified by --jars or spark.jars
```

Comment:
please leave a Spark JIRA ticket too, there are lots of Spark users maintain customized Spark versions